docs/README.md

```diff
@@ -12,7 +12,8 @@ This cluster definition examples demonstrate how to create a customized Docker E
 * [Kubernetes Walkthrough](kubernetes.md) - shows how to create a Kubernetes enabled Docker cluster on Azure
 * [Kubernetes Windows Walkthrough](kubernetes/windows.md) - shows how to create a hybrid Kubernetes Windows enabled Docker cluster on Azure.
 * [Kubernetes with GPU support Walkthrough](kubernetes.gpu.md) - shows how to create a Kubernetes cluster with GPU support.
-* [Kubernetes Kubernetes AAD integration Walkthrough](kubernetes.aad.md) - shows how to create a Kubernetes cluster with AAD as authentication provider.
+* [Kubernetes AAD integration Walkthrough](kubernetes.aad.md) - shows how to create a Kubernetes cluster with AAD as authentication provider.
```

Comment:
@bobbypage I just merged in a PR that fixes the AAD walkthrough link, so that's why the rebase. Thanks for this doc level-up!

Reply:
No problem, I've rebased, there shouldn't be any conflicts now.